src/wavesurfer.js

```diff
@@ -441,6 +441,39 @@ var WaveSurfer = {
     },
 
     /**
+     * Unmounts the wavesurfer instance from the page, but keeps the audio
+     * player in memory
+     */
+    unmount: function (selector) {
```

Comment:
No need to pass the selector since the current container is always referenced in `this.container`.